templates/index.html

```diff
@@ -369,13 +242,165 @@ <h2 class="mb-2 text-2xl">History</h2>
+                                "text-halign": "center",
+                                shape: "rectangle",
+                                height: "50px",
+                                width: (ele) => calcNodeWidth(ele.data("label")),
+                                color: function (ele) {
+                                    return getTextColor(ele.data("color"));
+                                },
+                                "font-size": "12px",
+                            },
+                        },
+                        {
+                            selector: "edge",
+                            style: {
+                                width: 3,
+                                "line-color": "data(color)",
+                                "target-arrow-color": "data(color)",
+                                "target-arrow-shape": "triangle",
+                                label: "data(label)",
+                                "curve-style": "unbundled-bezier",
+                                "line-dash-pattern": [4, 4],
+                                "text-background-color": "#ffffff",
+                                "text-background-opacity": 1,
+                                "text-background-shape": "rectangle",
+                                "font-size": "10px",
+                            },
+                        },
+                    ],
+                    layout: {
+                        name: "cose",
+                        fit: true,
+                        padding: 30,
+                        avoidOverlap: true,
+                    },
+                });
+            }
+
+
+
+
+
+            // Event listener for the form submission
+            document.getElementById("inputForm").addEventListener("submit", function (e) {
+                e.preventDefault(); // Prevent form submission
+                const userInput = document.getElementById("userInput").value;
+
+                // Add the loading class to start the animation
+                load.style.display = "block"; // show the load div
+                load.classList.add("loading");
+                document.getElementById("load").classList.add("loading");
+
+                fetch("/get_response_data", {
+                    method: "POST",
+                    headers: {
+                        "Content-Type": "application/json",
+                    },
+                    body: JSON.stringify({ user_input: userInput }),
+                })
+                    .then((response) => {
+                        if (!response.ok) {
+                            return response.text().then((text) => {
+                                throw new Error(text);
+                            });
+                        }
+                        return fetch("/get_graph_data", {
+                            method: "POST",
+                        });
+                    })
+                    .then((response) => {
```

Comment:
yeah. so just drop this bit then. and it will draw the graph with direction. it would be beautiful.

![image](https://github.com/yoheinakajima/instagraph/assets/10481417/ced4a600-eb38-48fa-84aa-f9dd7929e736)